src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -88,4 +118,7 @@ public DataSetDTO getDatasetDTO(Integer datasetId) {
             .collect(Collectors.toList());
     }
 
+    private boolean equalsProperty(Integer propOneKey, String propOneValue, Integer propTwoKey, String propTwoValue) {
+        return (propOneKey.equals(propTwoKey) && propOneValue.equalsIgnoreCase(propTwoValue));
+    }
```

Comment:
An alternative to this approach is to add a equals to DatasetProperty. I like apache commons `EqualsBuilder` for this. It would make sense to use dataset id, property key, and property value. Those three things should be unique or we have a different problem. See `User` for an example.